contracts/DKVDaggerHashimoto.sol

```diff
@@ -220,11 +284,19 @@ contract DecentralizedKVDaggerHashimoto is DecentralizedKV {
             mixData = _fnv256(i ^ uint256(hash0), mixData);
             uint256 parent = mixData % rows;
             uint256 kvIdx = parent + (startShardId << shardEntryBits);
-            bytes memory data = systemContract.unmaskWithEthash(kvIdx, maskedData[i]);
-            require(checkDaggerData(kvIdx, kvMap[idxMap[kvIdx]].hash, data), "invalid access proof");
+            bytes memory data = systemContract.unmaskWithEthash(
```

Comment:
do we need to change unmaskWithEthash like unmaskChunkWithEthash or change to unmaskChunkWithEthash?

Reply:
sorry, I don't know what you mean clearly.